src/mmtk.rs

```diff
@@ -401,7 +401,7 @@ impl<VM: VMBinding> MMTK<VM> {
     }
 
     /// The application code has requested a collection. This is just a GC hint, and
-    /// we may ignore it.
+    /// we may ignore it. Returns whether a GC was ran or not.
```

Comment:
```suggestion
    /// we may ignore it. If MMTk triggers a GC, this method will block the current thread
    /// and return true when the GC finishes. Otherwise, this method returns false immediately.
```

Reply:
Done.